Maddori.Apple/Maddori.Apple/Screen/MyBox/EditFeedbackFromMe/EditFeedbackFromMeViewController.swift

```diff
@@ -0,0 +1,117 @@
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupFeedbackType()
+        setupFeedbackKeyword()
+        setupFeedbackContent()
+        setupFeedbackStart()
+        hideEditFeedbackUntilLabel()
+        detectChangeOfFeedbackType()
+    }
+    
+    // MARK: - func
+    
+    private func setupFeedbackType() {
+        switch model.feedbackType {
+        case .continueType:
+            super.feedbackTypeButtonView.touchUpToSelectType(.continueType)
+        case .stopType:
+            super.feedbackTypeButtonView.touchUpToSelectType(.stopType)
+        }
+    }
+    
+    private func setupFeedbackKeyword() {
+        super.feedbackKeywordTextField.text = model.keyword
+        super.setCounter(count: model.keyword.count)
+    }
+    
+    private func setupFeedbackContent() {
+        super.feedbackContentTextView.text = model.info
+        super.feedbackContentTextView.textColor = .black100
+    }
+    
+    private func setupFeedbackStart() {
+        if let start = model.start {
+            super.feedbackStartSwitch.isOn = true
+            super.feedbackStartTextViewLabel.isHidden = false
+            super.feedbackStartTextView.isHidden = false
+            super.feedbackStartTextView.text = start
+            super.feedbackStartTextView.textColor = .black100
+        }
+    }
+    
+    private func hideEditFeedbackUntilLabel() {
+        super.editFeedbackUntilLabel.isHidden = true
+        super.feedbackDoneButtonView.snp.remakeConstraints {
+            $0.bottom.equalToSuperview()
+            $0.leading.trailing.equalToSuperview()
+            $0.height.equalTo(95)
+        }
+    }
+    
+    private func isTextInputChanged() -> Bool {
+        if super.feedbackContentTextView.text == model.info &&
+            super.feedbackStartTextView.text == model.start ?? TextLiteral.addFeedbackContentViewControllerStartTextViewPlaceholder &&
+            super.feedbackKeywordTextField.text == model.keyword {
+            return false
+        } else {
+            return true
+        }
+    }
+    
+    private func detectChangeOfFeedbackType() {
+        super.feedbackTypeButtonView.changeFeedbackType = { value in
+            if value == self.model.feedbackType {
+                self.isFeedbackTypeChanged = false
+            } else {
+                self.isFeedbackTypeChanged = true
+            }
+        }
+    }
+    
+    // MARK: - selector
+    
+    @objc override func willHideKeyboard(notification: NSNotification) {
+        UIView.animate(withDuration: 0.2, animations: {
+            super.feedbackDoneButton.transform = .identity
+        })
+        super.editFeedbackUntilLabel.isHidden = true
+    }
+    
+    // MARK: - extension
+    
+    override func textFieldDidChangeSelection(_ textField: UITextField) {
+        super.setCounter(count: textField.text?.count ?? 0)
+        super.checkMaxLength(textField: feedbackKeywordTextField, maxLength: Length.keywordMaxLength)
```

Comment:
p5;

요기도 `super` 부탁드립니다 !

```suggestion
        super.checkMaxLength(textField: super.feedbackKeywordTextField, maxLength: Length.keywordMaxLength)
```